LICENSE.md

```diff
@@ -0,0 +1,46 @@
+# License for branding assets
+
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
+Copyright (C) ?-2021 The Devicon team
+
+You are free to:
+Share — copy and redistribute the material in any medium or format
+Adapt — remix, transform, and build upon the material for any purpose, even commercially.
+
+Under the following terms:
+
+Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
+
+ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.
+
+Notices:
+
+You do not have to comply with the license for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation.
+No warranties are given. The license may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material.
+
+For more details:
+<https://creativecommons.org/licenses/by-sa/4.0/>
+<https://github.com/devicons/devicon/blob/master/branding_guidelines.md>
+
+## License for the source code
+
+The MIT License (MIT)
+
+Copyright (c) 2015-2021 The Devicon team
```

Comment:
```suggestion
Copyright (c) The Devicon team
```

Reply:
I don't think we need the end year, [just the year of first publication](https://www.copyrightlaws.com/copyright-symbol-notice-year/#:~:text=There%20are%20three,Mary%20Clark%C2%A0%202021).
So `Copyright (c) 2015 The Devicon team` would work just fine.


If you're afraid of the copyright expiring, that's nothing to worry about, [since it only expires 70 years after the authors death](https://www.copyright.gov/help/faq/faq-duration.html#:~:text=As%20a%20general,whichever%20expires%20first.).

Fun fact related to this: The Eiffel Tower is illegal to film at night to film, since the lights are copyrighted. [Or is it?](https://www.reddit.com/r/curiousvideos/comments/760p6r/comment/doarxkx/?utm_source=share&utm_medium=web2x&context=3)